src/commands/cli/release/build.ts

```diff
@@ -189,6 +189,10 @@ export default class build extends SfCommand<void> {
     await this.exec('npx yarn-deduplicate');
     // Run an install with deduplicated dependencies (with scripts)
     await this.exec('yarn install');
+    // Generate a new readme with the latest dependencies
+    await this.exec(
+      'yarn oclif readme --no-aliases --repository-prefix "<%- repo %>/blob/<%- version %>/<%- commandPath %>"'
```

Comment:
Would you prefer to set this in `oclif.repositoryPrefix` in the package.json? That way it doesn't change if someone were to, for whatever reason, run `readme` manually in the sf repo

Reply:
Oh yea, good call. Let me test that.

Reply:
Huh, that's not going to work... It is looking at each plugin's `package.json` for that setting. 

This makes which option gets precedence [here](https://github.com/oclif/oclif/blob/149ef6ef296cdebc49793679a95dddc72d2debfd/src/commands/readme.ts#L216) more tricky. My initial thought was "If you pass the `--repository-prefix` flag, use it. That way it can overwrite the value in the `package.json`. I think that this is still ideal at the plugin level... Otherwise if you needed to use a different value for the template, you'd potentially have to update a lot of different plugins.

But if you run this from "the CLI level", the order should be:
1. Look at the plugin's `package.json` for `oclif.repositoryPrefix` 
    1. If it's set there, there is probably a good reason for it. 
3. Then use the `--repository-prefix` value
4. Then use the default

Thoughts? We can also move this work to a new ticket.

Reply:
Yeah I think that makes sense even though flags typically override configurations. Might just be worth noting in the flag's description.

I'm fine solving that in separate WI if you prefer

Reply:
Yea, lets leave this as is for now and I can create a new ticket

Reply:
I created W-14208895 for the additional work